source/manuals/programming-languages/python/python.html.md.erb

```diff
@@ -35,119 +37,81 @@ Couple this with the fact that much of the time GDS developers are coding web ap
 have to deal with nested `JSON` objects, ORM model definitions/ queries, and error/ url
 strings and this convention begins to show its age.
 
-```
-Do not do
-
-if not models.Address.query(
-    models.Address.street_address_line_1
-    == user['address']['street_address_line_1']
-):
-    pass
-
-
-Do
-
-if not models.Address.query(models.Address.street_address_line_1 == user['address']['street_address_line_1']):
-    pass
-```
-
 
 ## Linting
 
 
-### Flake8
-
-This manual advises the use of the [Flake8][] command line checker as an all in one lint, codestyle and complexity checker.
+### Ruff
 
-#### How to use Flake8
+This manual advises the use of the [Ruff][] command line checker as an all in one lint,
```

Comment:
Just spotted you're missing a reference link here. Ruff also linked to on line 73

Reply:
Good spot! fixed and pushed. wonder if theres any md linting 🤔